src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -395,7 +395,7 @@ private void postChange( final IAEItemStack diff, final IActionSource src )
 
 	private void markDirty()
 	{
-		this.getCore().markDirty();
+		this.getCore().saveChanges();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getCore()" can return null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2259)